tw-gaffer-jta-starter/src/main/resources/META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports

```diff
@@ -0,0 +1 @@
+com.transferwise.common.gaffer.starter.GafferJtaAutoConfiguration
```

Comment:
Since this version will only support Spring Boot 3 then we can already clean up the old auto configuration way as well by removing `spring.factories` file.

Reply:
Yep, will do.